examples/advanced_agents/sandbox_agent/src/nat_sandbox_agent/tools/sandbox/executor.py

```
@@ -0,0 +1,65 @@
+    def truncate(self, text: str) -> str:
+        """Truncate output to maximum length."""
+        return truncate_output(text, self.max_output_chars)
+
+    async def list_generated_files(self) -> list[str]:
+        """List files in the output directory using shell command."""
+        try:
+            result = await self.sandbox.run_command("ls -1 /workspace/output")
+            if result.success:
+                files = [f.strip() for f in result.stdout.strip().split("\n") if f.strip()]
+                return [f"/workspace/output/{f}" for f in files]
+            return []
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)